apps/production/prod-rucio-daemons.yaml

```diff
@@ -29,7 +29,7 @@ replicaRecoverer:
 
 reaper:
   threads: 4
-  includeRses: "reaper=True"
+  includeRses: "cms_type=real&reaper=True\\rse_type=TAPE\\tier=1\\tier=0\\T2_CH_CERN"
```

Comment:
This will make the default reapers stop working on test and temp RSEs. I'll change it.